server-spi-private/src/main/java/org/keycloak/migration/migrators/MigrateTo12_0_0.java

```diff
@@ -0,0 +1,28 @@
+package org.keycloak.migration.migrators;
+
+import java.util.Objects;
+
+import org.keycloak.migration.ModelVersion;
+import org.keycloak.models.AccountRoles;
+import org.keycloak.models.KeycloakSession;
+
+public class MigrateTo12_0_0 implements Migration {
```

Comment:
Should we add a test for this to [`MigrationTest`](https://github.com/keycloak/keycloak/blob/c403aa49f724fec793e3babc3de4bf4e9753bf24/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/migration/MigrationTest.java)?

Reply:
why not

Reply:
Done!